app/javascript/components/modal/Modal.vue

```diff
@@ -97,6 +101,15 @@ export default {
 
     themeTitle () {
       return this.data.modal && this.data.modal.themeTitle ? this.data.modal.themeTitle : false
+    },
+
+    backgroundStyles () {
+      const { image, image_position } = this.data.modal
+
+      return {
+        'background-image': `url(${image})`,
+        'background-position': `${image_position ? image_position : '0% 0%'}`
```

Comment:
Why not `image_position ? image_position : '0% 0%'`?

Reply:
You can also use _camelCase_ in style objects, so you can simplify to backgroundImage and backgroundPosition (without the need to turn them into hyphen-separated strings)